build_extra/rust/BUILD.gn

```diff
@@ -69,3 +69,25 @@ rust_component("cfg_if") {
   source_root =
       "$crates/registry/src/github.com-1ecc6299db9ec823/cfg-if-0.1.4/src/lib.rs"
 }
+
+rust_component("clap") {
+  source_root =
+      "$crates/registry/src/github.com-1ecc6299db9ec823/clap-2.32.0/src/lib.rs"
+  extern = [ ":unicode_width", ":textwrap", ":bitflags" ]
+}
```

Comment:
> "Clap is a simple-to-use, efficient, and full-featured library"

I didn't explain this well in the issue, but I just want basic parsing in the Rust layer - and then pass the majority of argv off to JS where there will be further flag parsing. I think [getopts](https://github.com/rust-lang-nursery/getopts) would work for this?